cli/src/maintenance.rs

```diff
@@ -692,6 +709,21 @@ impl SolidoState {
         None
     }
 
+    /// If there is a validator ready for removal, try to remove it.
+    pub fn try_sell_anker_rewards(&self) -> Option<(Instruction, MaintenanceOutput)> {
+        if self.anker_state.ust_reserve_balance > MicroUst(0) {
```

Comment:
This is the condition for _sending_ the rewards, not selling. For selling we need to compute the SOL value of the stSOL in the reserve according to Solido's exchange rate, and if that exceeds the bSOL supply, then we sell the rewards.

Reply:
Yeah, that's true. Thanks! I got sidetracked with another PR to setup a token pool, Will come back to this.